two-factor.php

```diff
@@ -47,34 +47,32 @@ function wpcom_vip_should_force_two_factor() {
 		return false;
 	}
 
-	// Don't force 2FA for OneLogin SSO
-	if ( function_exists( 'is_saml_enabled' ) && is_saml_enabled() ) {
-		return false;
-	}
-
-	// Don't force 2FA for SimpleSaml
-	if ( function_exists( '\HumanMade\SimpleSaml\instance' ) && \HumanMade\SimpleSaml\instance() ) {
+	// Allow custom SSO solutions
+	if ( wpcom_vip_use_custom_sso() ) {
 		return false;
 	}
 
 	return true;
 }
 
-function wpcom_vip_is_jetpack_authorize_request() {
```

Comment:
Accidentally deleted this function?

Reply:
🤔

Reply:
I have no idea why it's red. That function is still there.

Reply:
https://github.com/Automattic/vip-go-mu-plugins/blob/5a5059e1a110a7d41dcecbe4280afe7e377e9f29/two-factor.php#L83-L90

Reply:
I don't think so https://github.com/Automattic/vip-go-mu-plugins/blob/5a5059e1a110a7d41dcecbe4280afe7e377e9f29/two-factor.php

Reply:
@justnorris that's a different function. `wpcom_vip_is_jetpack_authorize_request` vs `wpcom_vip_is_jetpack_authorize_rest_request`.

Reply:
Fixed